src/pages/index.tsx

```diff
@@ -80,26 +77,15 @@ const Home: NextPage = React.forwardRef(function Home() {
   const chartRef = useRef(null as null | HTMLDivElement);
   const chartContainerRef = useRef(null as null | HTMLDivElement);
 
-  function chartTimeChanged(timeRange: ChartTimeRange) {
-    setChartTimeRange(timeRange);
-  }
-
+  // This effect will run only once when this component is mounted
```

Comment:
@Ceraph1216 instead of one `useEffect` call for each store, we can combine all the ones that only need to run once into one useEffect. The idea here being that a `useEffect` callback will re-fire if any of its dependencies (values passed into the second parameter array) change. Since all dependencies here are static function references, this will only fire once.